resources/views/platform/components/game/add-to-list.blade.php

```diff
@@ -0,0 +1,58 @@
+@props([
+    'user' => null,
+    'gameId' => 0,
+    'type' => 'play',
+])
+
+<?php
+use App\Site\Enums\Permissions;
+
+$addButtonTooltip = __('user-game-list.' . $type . '.add');
+$removeButtonTooltip = __('user-game-list.' . $type . '.remove');
+$addVisibility = '';
+$removeVisibility = '';
+$onLists = !empty($user) ? getUserGameListsContaining($user, $gameId) : [];
+
+$buttonTooltip = '';
+if (in_array($type, $onLists)) {
+    $addVisibility = 'hidden';
+    $buttonTooltip = $removeButtonTooltip;
+} else {
+    $removeVisibility = 'hidden';
+    $buttonTooltip = $addButtonTooltip;
+}
+?>
+
+@if (!empty($user))
+<button id='{{ $type }}-list-button' class='btn' type='button' title='{{ $buttonTooltip }}'
+        onClick="toggleListItem('{{ $type }}')">
+    <div class="flex items-center gap-x-1">
+        <div id='add-to-{{ $type }}-list' class="md:float-left w-[18px] h-[18px] {{ $addVisibility }}">
```

Comment:
nit: I believe `md:float-left` is inert.

Reply:
It probably is. I'm sure I copied that from somewhere. css isn't my strong suit.

Reply:
That's okay. SQL isn't my strong suit and I still have a lot to learn about PHP. I'm glad we can complement each other well.